sdk/src/main/scala/com/akkaserverless/javasdk/impl/AnySupport.scala

```diff
@@ -389,6 +389,11 @@ class AnySupport(descriptors: Array[Descriptors.FileDescriptor],
           scalaPbMessage.toByteString
         )
 
+      case null =>
+        throw SerializationException(
+          s"Don't know how to serialize object of type null. Try passing a protobuf, using a primitive type, or using a type annotated with @Jsonable."
+        )
```

Comment:
Got here when reflect-calling old void `@EventHandlers` so won't happen in the future but for good measure.